.azure-devops/create-release.yml

```diff
@@ -3,6 +3,13 @@ pr: none
 trigger: none
 
 parameters:
+- name: vmImage
+  type: string
+  default: 'ubuntu-18.04'
```

Comment:
Note that ubuntu 16.04 is going away soon as an agent image option therefore the new pipeline updates must work on at least ubuntu 18.04

Reply:
This chunk of code should resole DNS resolution issues on ubuntu 18.04 
![image](https://user-images.githubusercontent.com/28658112/129799912-c5cfbbee-54ad-4a53-aa2b-2afe1a95e945.png)

Reply:
So basically, I should remove the 16.04 option and also add this snippet to all the YAMLs?

Reply:
The snippet should be added prior to running tests, rather than on all yamls.